fe/fe-core/src/main/java/org/apache/doris/nereids/properties/ChildrenPropertiesRegulator.java

```diff
@@ -464,6 +467,9 @@ public Boolean visitPhysicalNestedLoopJoin(PhysicalNestedLoopJoin<? extends Plan
     @Override
     public Boolean visitPhysicalProject(PhysicalProject<? extends Plan> project, Void context) {
         // do not process must shuffle
+        if (children.get(0).getPlan() instanceof PhysicalDistribute) {
```

Comment:
if (children.get(0).getPlan() instanceof PhysicalDistribute && project.getOutput().size()  <= children.get(0).getPlan().getOutput().size()) { ...}
how about this?

Reply:
too many check deps on project must under distribute, so we forbid it now until we found a way to remove those check